test/testExtractScripts.js

```diff
@@ -0,0 +1,51 @@
+const assert = require('assert');
+const fs = require('fs');
+const tmp = require('tmp');
+
+const extractScriptsFromHTML = require('../src/extractScripts.js');
+
+// IMPORTANT: if file naming convention changes in file, change this function.
```

Comment:
Maybe add a test that will fail if naming convention doesn't match, and that could be a good place to document what the convention is and which code is affected.

Reply:
The test in testExtractScripts.js under `it('should return correct list of files', function() {` will fail if the naming convention doesn't match since the list of files returned by the tested function will not be the same as the one generated in `expectedResults()`. I'll add a comment by the `assert` that will highlight this again though.